CHANGELOG.md

```diff
@@ -1,10 +1,15 @@
+# dbt_google_ads_source v0.10.0
+
+## Under the Hood:
```

Comment:
Can we also include that we removed the test as well.

Additionally with the test being removed I think we can remove [this macro](https://github.com/fivetran/dbt_google_ads_source/blob/update_deps_versions/macros/regexp_instr.sql), right?

Reply:
updated, and good catch, removed!